modules/installation-methods-for-different-architectures.adoc

```diff
@@ -21,6 +21,6 @@ a|Operator (preferred), Helm charts, or `roxctl` CLI (not recommended)
 
 |s390x ({ibm-z} and {ibm-linuxone})
 
-|aarch64 ({arm64})
+|aarch64 (`arm64`)
```

Comment:
Should it be "Aarch 64 (ARM64)" because we are referring to the architecture? 

https://redhat-documentation.github.io/supplementary-style-guide/#aarch64
https://redhat-documentation.github.io/supplementary-style-guide/#arm64